build/Makefile

```diff
@@ -246,7 +248,8 @@ test-e2e-integration: $(ensure-build-image)
 	echo "Starting e2e integration test!"
```

Comment:
Add `FEATURE_GATES ?= ALL_FEATURE_GATES` here or above. So we can change them for specific